src/handlers.ts

```diff
@@ -0,0 +1,23 @@
+import * as daylightSavingsChangeReminder from './daylightSavingsChangeReminder';
+import * as leaveReminder from './leaveReminder';
+import { Notifier } from './notifier';
+import * as publicHolidayReminder from './publicHolidayReminder';
+import type { NotifierResult } from './types';
+
+export const weeklyReport = async (): Promise<NotifierResult> => {
```

Comment:
🐘 Ideally it would be good to use the types from `@types/aws-lambda` but that might be a bit of extra work

Reply:
I think it's worth adding them for completeness